spec/factories.rb

```diff
@@ -0,0 +1,22 @@
+require 'factory_helpers'
+
+FactoryBot.define do
+  
+  factory :user do
+    name { FactoryHelpers.random_string }
```

Comment:
you should decide if you want to use everywhere:
```ruby
field { value }
# or
field value
```

I would recommend using the first one, adding `{ }` to all values